### ISSUE TITLE
Add term proximal radioulnar joint dislocation

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10410]
**Preferred term label:**
Proximal radioulnar joint dislocation

**Synonyms**

**Definition (free text, please give PubMed ID)**
The proximal radioulnar translocation means that both the radius and ulna are dislocated from the humerus. PMID: 20660242

**Parent term (use hpo.jax.org/app)**
Radioulnar dislocation HP:0006439

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Monteggia fractuur

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 

Adding new term: 
- ID: HP:6000890
- label: Proximal radioulnar joint dislocation
- parents: [Radioulnar dislocation (HP:0006439)](https://hpo.jax.org/app/browse/term/HP:0006439)
- pmid: PMID:20660242
- def: he proximal radioulnar translocation means that both the radius and ulna are dislocated from the humerus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10410
- synonyms:
